app/src/main/java/org/fossasia/phimpme/utilities/Urls.java

```diff
@@ -0,0 +1,7 @@
+package org.fossasia.phimpme.utilities;
+
+/** Created by @codedsun on 19/Oct/2019 */
+public class Urls {
+
+  public static final String IMGUR_BASE_URL = "https://api.imgur.com";
```

Comment:
Not useful as it is not reused

Reply:
It's used while creating retrofit client

Reply:
https://en.wikipedia.org/wiki/Reuse

This is an interesting read on this topic

Reply:
@iamareebjamal We have one place for keeping all the API's URL.

Reply:
i will keep all the other accounts' url api endpoints here, if you want me to remove the file and use that directly in activity or something i will do that.

Reply:
That's a solid anti-pattern